src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -49,13 +49,14 @@ case class ClusterError(errorMessage: String,
 
 // The cluster itself
 // Also the API response for "list clusters" and "get active cluster"
-case class Cluster(clusterName: ClusterName,
-                   googleId: UUID,
+case class Cluster(id: Long = 0, // DB AutoInc
```

Comment:
These changes to `Cluster` will need to be reflected in Swagger, since Cluster's json representation is returned in Leo APIs.

We should also double check that this is not a breaking change. I don't think so because we're not removing any fields -- just adding new fields (`id`) and making existing fields optional (`googleId`, `operationName`). Sound right?

Reply:
Sounds right. As far as the Swagger changes, I'm not adding `id` to `Cluster` on `api-docs.yaml` as I couldn't see a reason how to expose it. If this changes during Phase 2, then we can add it. Does that sound good?

Reply:
I think we should probably still add `id` to the Cluster model here: https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/swagger/api-docs.yaml#L560

Since the `Cluster` case class is deserialized directly to json, it _will_ be exposed to the user in API responses.